groups/iam/module-iam-policies/test-data-generator-stack-terraform-destroy.tf

```diff
@@ -0,0 +1,53 @@
+data "aws_iam_policy_document" "test_data_generator_stack_terraform_destroy" {
+
+  statement {
+    sid       = "DestroyPolicyAllResources"
+    effect    = "Allow"
+    resources = ["*"]
+    actions = [
+      "ec2:DescribeImages",
+      "ecs:DeregisterTaskDefinition",
+    ]
+  }
+
+  statement {
+    sid    = "DestroyPolicyListedResources"
+    effect = "Allow"
+    resources = [
+      "arn:aws:ssm:eu-west-2:${var.aws_account_id}:parameter/test-data-generator-devops1/secret-mongo-url",
+      "arn:aws:iam::${var.aws_account_id}:role/test-data-generator-devops1-ecs-instance-role",
+      "arn:aws:iam::${var.aws_account_id}:role/test-data-generator-devops1-ecs-service-role",
+      "arn:aws:iam::${var.aws_account_id}:role/test-data-generator-devops1-ecs-task-execution-role",
+      "arn:aws:autoscaling:eu-west-2:${var.aws_account_id}:autoScalingGroup:*:autoScalingGroupName/test-data-generator-devops1-ecs-asg",
+      "arn:aws:autoscaling:eu-west-2:${var.aws_account_id}:launchConfiguration:*:launchConfigurationName/test-data-generator-devops1*",
+      "arn:aws:ecs:eu-west-2:${var.aws_account_id}:service/test-data-generator-devops1-cluster/devops1-test-data-generator",
+      "arn:aws:ecs:eu-west-2:${var.aws_account_id}:cluster/test-data-generator-devops1-cluster",
+      "arn:aws:elasticloadbalancing:eu-west-2:${var.aws_account_id}:listener/net/test-data-devops1-lb/*/*",
+      "arn:aws:elasticloadbalancing:eu-west-2:${var.aws_account_id}:listener/app/test-data-devops1-lb/*/*",
+      "arn:aws:elasticloadbalancing:eu-west-2:${var.aws_account_id}:loadbalancer/app/test-data-devops1-lb/*",
+      "arn:aws:elasticloadbalancing:eu-west-2:${var.aws_account_id}:loadbalancer/net/test-data-devops1-lb/*",
+      "arn:aws:elasticloadbalancing:eu-west-2:${var.aws_account_id}:targetgroup/devops1-test-data-generator/*",
+      "arn:aws:ec2:eu-west-2:${var.aws_account_id}:security-group/*",
```

Comment:
I'd suggest this one is too open

Reply:
Yes, it's a difficult one to restrict because the ARN contains the security group ID rather than the name. I'll see what I can do with conditions.